index.html

```diff
@@ -247,11 +247,15 @@ <h2>Top Security</h2>
                         online daily to run your daily
                         transactions our mobile 
                         app is ready for download at no cost
-                    </p>
+                    
                 </div>
             </div>
             <a href="#"><button>Download App</button></a>
         </div>
+        <!-- <div class="how">
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Impedit officia provident consequuntur praesentium neque quam molestiae, ipsam non voluptatum consectetur debitis harum itaque, obcaecati nesciunt quod suscipit unde porro in.</p>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel esse quos nemo rem cumque aspernatur dolorem amet animi est ipsam mollitia, voluptatum placeat minima delectus earum vero repudiandae aperiam. Voluptatum.</p>
+        </div> -->
```

Comment:
Always remove unused code . It makes your code clean

Reply:
Good job! but it's highly recommended to suggest instead of ordering. So please try to use a respective sentence in which the code author will be respected.